src/main/java/goodgenerator/loader/ComponentAssemblyLineRecipeLoader.java

```diff
@@ -87,8 +82,8 @@ private static void generateAssemblerRecipes() {
                     }
 
                     int tier = info.getRight();
-                    int energy = (int) Math.min(Integer.MAX_VALUE - 7, (GT_Values.V[tier] - (GT_Values.V[tier] >> 4)));
-
+                    int energy =
+                            (int) Math.min(Integer.MAX_VALUE - 7, (GT_Values.V[tier] - (GT_Values.V[tier] >> 4)) * 2);
```

Comment:
why do you multiply by 2?

Reply:
Changed to `GT_Values.VP[tier - 1]`. LuV components use IV energy, so tier - 1 makes sense